### PR TITLE
Remove data_feed data_show and feeds

### PR DESCRIPTION
**Pull request by bmatherly**
I am unsure if these are still used or if the comment should be changed to not reference "data_feed":
```
 * \properties \em global_feed a flag to indicate whether this tractor feeds to the consumer or stops here
 * \properties \em global_queue is something for the data_feed functionality in the core module
 * \properties \em data_queue is something for the data_feed functionality in the core module
```
https://github.com/mltframework/mlt/blob/83b6b806c41af779cba5c6763b9bda1cfb5f51cb/src/framework/mlt_tractor.h#L37

### CODE REVIEW

**Comment by ddennedy** [comment]
"global_feed", "global_queue", and "data_queue" are all related to the removed data_feed filter. Basically lines 567 - 586 in `mlt_tractor.c` can be removed.

**Comment by bmatherly** [comment]
> "global_feed", "global_queue", and "data_queue" are all related to the removed data_feed filter. Basically lines 567 - 586 in mlt_tractor.c can be removed.

Done. This even cleans up the XML a little bit.

File: src/modules/xml/mlt-xml.dtd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='utf-8'?>
 
-<!-- MLT XML DTD v0.2.0 -->
```
Comment (ddennedy) [inline src/modules/xml/mlt-xml.dtd:3]: I am thinking perhaps we should change the DTD version to a major and minor only that reflects the MLT version. We do not need to increment it with each minor version of MLT, but anytime we break the schema by changing it in a non-backwards compatible way, it should coincide with a major new version of MLT. And anytime we add something in a backwards compatible way or fix the schema bump the minor version. The XML interface is nearly as important as the ABI. See how in Shotcut I am now using the MLT major version reflected in the MLT XML to check compatibility.

Comment (bmatherly) [reply src/modules/xml/mlt-xml.dtd:3]: I also need to add chain and link types to the dtd before the official release.

Shall I change the DTD version to 7.0.0?

Comment (ddennedy) [reply src/modules/xml/mlt-xml.dtd:3]: yes, change it to 7.0.0

Comment (bmatherly) [reply src/modules/xml/mlt-xml.dtd:3]: Done

**Pull request merged**